packages/ra-core/src/auth/Authenticated.tsx

```diff
@@ -36,17 +30,15 @@ export interface AuthenticatedProps {
  *         </Admin>
  *     );
  */
-const Authenticated = (props: AuthenticatedProps) => {
-    const {
-        authParams,
-        children,
-        location, // kept for backwards compatibility, unused
-        ...rest
-    } = props;
+export const Authenticated = (props: AuthenticatedProps) => {
+    const { authParams, children } = props;
     useAuthenticated({ params: authParams });
     // render the child even though the useAuthenticated() call isn't finished (optimistic rendering)
     // the above hook will log out if the authProvider doesn't validate that the user is authenticated
-    return cloneElement(children, rest);
+    return children;
```

Comment:
i'd prefer that you return `<>{children}</>`, this makes TypeScript happier. 

also, please type `children` as `ReactNode` instead of `ReactElement` to allow multiple children, e.g. 

```jsx
const Foo = () => (
  <Authenticated>
    <Bar />
    <Baz />
  </Authenticated>
);
```